numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -2766,17 +2828,27 @@ array_correlate2(PyObject *NPY_UNUSED(dummy), PyObject *args, PyObject *kwds)
 static PyObject *
 array_arange(PyObject *NPY_UNUSED(ignored), PyObject *args, PyObject *kws) {
     PyObject *o_start = NULL, *o_stop = NULL, *o_step = NULL, *range=NULL;
-    static char *kwd[]= {"start", "stop", "step", "dtype", NULL};
+    PyObject *like = NULL;
+    PyObject *array_function_result = NULL;
+    static char *kwd[]= {"start", "stop", "step", "dtype", "like", NULL};
```

Comment:
```suggestion
    static char *kwd[] = {"start", "stop", "step", "dtype", "like", NULL};
```

Reply:
Done in https://github.com/numpy/numpy/pull/16935/commits/e94e92e175b0a2705f7b5b2e4b9958ba136b051b .